pages/volunteer.liquid

```diff
@@ -123,37 +123,35 @@ eleventyNavigation:
 
       <div>
         <div class="bio-with-pic">
-          <img class="bio-pic" src="/assets/images/volunteers/wendy-pic.jpg" alt="Wendy Holley headshot">
-          <h3>Wendy Holley</h3>
+          <img class="bio-pic" src="/assets/images/volunteers/MorganHaworth.jpg" alt="Morgan Haworth headshot">
+          <h3>Morgan Haworth</h3>
           <h4>Board Member</h4>
           <div class="badge-container">
             <div class="gwc-badge instructor">{I}</div>
-            <div class="gwc-badge ta">[TA]</div>
-            <div class="gwc-badge organizer">&lt;O&gt;</div>
           </div>
         </div>
       </div>
 
       <div>
         <div class="bio-with-pic">
-          <img class="bio-pic" src="/assets/images/volunteers/autumn.jpg" alt="Autumn Rodeheaver headshot">
-          <h3>Autumn Rodeheaver</h3>
+          <img class="bio-pic" src="/assets/images/volunteers/wendy-pic.jpg" alt="Wendy Holley headshot">
+          <h3>Wendy Holley</h3>
           <h4>Board Member</h4>
           <div class="badge-container">
+            <div class="gwc-badge instructor">{I}</div>
+            <div class="gwc-badge ta">[TA]</div>
             <div class="gwc-badge organizer">&lt;O&gt;</div>
           </div>
         </div>
       </div>
 
       <div>
         <div class="bio-with-pic">
-          <img class="bio-pic" src="/assets/images/volunteers/biancaz.jpg" alt="Bianca Zongrone Jefferson headshot">
-          <h3>Bianca Zongrone Jefferson</h3>
+          <img class="bio-pic" src="/assets/images/volunteers/tera.jpg" alt="Tera Maher headshot">
+          <h3>Tera Maher</h3>
           <h4>Board Member</h4>
           <div class="badge-container">
-            <div class="gwc-badge ta">[TA]</div>
-            <div class="gwc-badge speaker">"S"</div>
-            <div class="gwc-badge organizer">&lt;O&gt;</div>
+            <div class="gwc-badge instructor">{I}</div>
```

Comment:
I think Tera has been a speaker rather than instructor?  I'm happy to be corrected.

Reply:
I thought she lead a Code Crush thing one day, or is that a speaker?